web/src/lib/components/shared-components/side-bar/side-bar.svelte

```diff
@@ -69,9 +73,30 @@
 			};
 		}
 	};
+
+	const handleResize = () => {
+		if (innerWidth > 768) {
+			isCollapsed = false;
+		} else {
+			isCollapsed = true;
+		}
+	};
+
+	onMount(() => {
+		handleResize();
+		window.addEventListener('resize', handleResize);
+	});
 </script>
 
-<section id="sidebar" class="flex flex-col gap-1 pt-8 pr-6 bg-immich-bg dark:bg-immich-dark-bg">
+<section
+	id="sidebar"
+	on:mouseover={() => (innerWidth >= 430 ? (isCollapsed = false) : null)}
+	on:focus={() => null}
```

Comment:
You probably added this empty event listener to stop a11y from complaining, but it would be preferred to correctly handle this situation by showing the extended sidebar on focus

Reply:
Done